libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/tabs/workflowTab.tsx

```diff
@@ -37,6 +43,12 @@ export const workflowTab = (intl: IntlShape, dispatch: AppDispatch): TemplatePan
       description: 'Button text to create workflow from this template',
     }),
     primaryButtonOnClick: () => {
+      LoggerService().log({
+        level: LogEntryLevel.Trace,
+        area: 'Templates.workflowTab',
```

Comment:
Not blocking, but I thought we are going to use same area for overview tabs both summary and workflow as Templates.overviewTab